ya-runtime-basic-auth/example_config/plugins/ya-runtime-erigon.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "name": "erigon",
```

Comment:
When I changed to `ya-runtime-erigon` here and the corresponding `exeunit-name` in the `preset.json`, golemsp added [this entry](https://github.com/golemfactory/yagna-service-erigon/blob/9136a5c54464222126a2b0fb622c458477769b0a/ya-runtime-basic-auth/example_config/ya-provider/presets.json#L29-L37) and pushed my "erigon" entry to the bottom of the presets file.

Reply:
The hypothesis is that preset's name should be the same as exeunit name. ❓

Reply:
See Yagna issue golemfactory/yagna#1701